cli/exchange/nmp.go

```diff
@@ -118,6 +119,27 @@ func NMPAdd(org, credToUse, nmpName, jsonFilePath string, appliesTo, noConstrain
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("The node management policy has no constraints which might result in the management policy being deployed to all nodes. Please specify --no-constraints to confirm that this is acceptable."))
 	}
 
+	// Validate the AgentAutoUpgrade job if it was defined
+	if nmpFile.AgentAutoUpgradePolicy != nil {
+
+		// Ensure that a manifest was specified
+		fullManifest := nmpFile.AgentAutoUpgradePolicy.Manifest
+		if fullManifest == ""{
+			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("An AgentAutoUpgrade job was defined, but a manifest was not defined. Please specify a manifest that is stored in the CSS before attempting to add an NMP with an AgentAutoUpgrade job."))
```

Comment:
Do we refer to this as a AgentAutoUpgrade job in other places? Seems like it is a NMP with an Agent Upgrade Policy.  The usage of job seems odd to me... and line 139 too

Reply:
I used that term a lot in the documentation since you define a node management policy to define the job that will be executed, but I see your point. I may edit the docs as well if we decide to go with the term Agent Upgrade Policy within a Node Management Policy